src/NUnitFramework/framework/Assert.cs

```diff
@@ -74,19 +75,21 @@ public partial class Assert
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static new bool Equals(object a, object b)
         {
-            throw new InvalidOperationException("Assert.Equals should not be used for Assertions");
+            throw new InvalidOperationException("Assert.Equals should not be used for Assertions, use Assert.AreEqual(...) instead.");
         }
 
         /// <summary>
+        /// DO NOT USE! Use Assert.AreEqual(...) instead.
         /// override the default ReferenceEquals to throw an InvalidOperationException. This 
         /// implementation makes sure there is no mistake in calling this function 
         /// as part of Assert. 
         /// </summary>
         /// <param name="a"></param>
         /// <param name="b"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static new void ReferenceEquals(object a, object b)
         {
-            throw new InvalidOperationException("Assert.ReferenceEquals should not be used for Assertions");
+            throw new InvalidOperationException("Assert.ReferenceEquals should not be used for Assertions, use Assert.AreEqual(...) instead.");
```

Comment:
Recommending `AreEqual` isn't right. I'd be okay with `AreSame` or nothing.

BTW, it probably doesn't make a different, but shouldn't this method return a bool to be consistent?